Student_BCOs/XiyingDing_BCO_000524.json

```diff
@@ -1,27 +1,63 @@
 {
     "bco_id": "http://biocomputeobject.org/BCO_000524",
-    "checksum": "893227528679a28274e102aa60ef67b64dbab5f8e064060edc5ba9f6fa4b464d",
+    "checksum": "18ba0128bf0e79405419828fb948b34b27c6db1effd0c06cf213635786a25070",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
-        "name": "TAP: a targeted clinical genomics pipeline for detecting transcript variants using RNA-seq data",
+        "name": "Detection of transcript variation by analysis of RNA-seq data using targeted clinical genomics pipeline TAP",
         "version": "0.4.2",
         "license": "http://creativecommons.org/licenses/by/4.0/",
         "created": "2019-10-07T22:19:18.384004",
-        "modified": "",
+        "modified": "2019-10-18T02:29:31.360979",
         "contributors": [
             {
-                "name": "",
+                "name": "Xiying Ding",
                 "orcid": "",
-                "affiliation": "",
+                "affiliation": "George Washington University",
                 "contribution": [
                     "createdBy"
                 ],
                 "email": "xzd@gwu.edu"
+            },
+            {
+                "name": "Readman Chiu",
+                "orcid": "https://orcid.org/0000-0002-2215-5535",
+                "affiliation": "Canada’s Michael Smith Genome Sciences Centre, BC Cancer Agency",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy", please also include the "derivedFrom" contribution type for publication authors.